server/app/views/components/FieldWithLabel.java

```diff
@@ -283,10 +301,16 @@ public ContainerTag getContainer() {
     }
 
     boolean hasFieldErrors = !fieldErrors.isEmpty() && showFieldErrors;
+    fieldTag.condAttr(hasFieldErrors || hasQuestionErrors, "aria-invalid", "true");
+
+    var allAriaDescribedByIds = new ArrayList<String>(ariaDescribedByIds);
     if (hasFieldErrors) {
-      fieldTag.attr("aria-invalid", "true");
-      fieldTag.attr("aria-describedBy", fieldErrorsId);
+      allAriaDescribedByIds.add(0, fieldErrorsId);
     }
+    fieldTag.condAttr(
+        !allAriaDescribedByIds.isEmpty(),
```

Comment:
Is a blank value acceptable for `aria-describedBy`? If so, we could unconditionally set this.